docs/source/asr/configs.rst

```diff
@@ -256,3 +291,104 @@ For example, a decoder config corresponding to the encoder above would look like
+The encoder parameters include details about the Citrinet-C encoder architecture, including how many
+filters are used per channel (C). The Citrinet-C configuration is a shortform notation for Citrinet-21x5xC, such that B = 21 and R = 5 are the default and should generally not be changed.
+
+To use Citrinet instead of QuartzNet, please refer to the ``citrinet_512.yaml`` configuration found inside the ``examples/asr/conf/citinet`` directory. While the configs for Citrinet and QuartzNet are similar, we note the additional flags used for Citrinet below
+
++-------------------------+------------------+---------------------------------------------------------------------------------------------------------------+---------------------------------+
+| **Parameter**           | **Datatype**     | **Description**                                                                                               | **Supported Values**            |
++=========================+==================+===============================================================================================================+=================================+
+| :code:`se`              | bool             | Whether to apply squeeze-and-excitation mechanism or not.                                                     | :code:`true` or :code:`false`   |
++-------------------------+------------------+---------------------------------------------------------------------------------------------------------------+---------------------------------+
+| :code:`se_context_size` | int              | SE context size. -1 means global context.                                                                     | :code:`-1` or :code:`+ve int`   |
++-------------------------+------------------+---------------------------------------------------------------------------------------------------------------+---------------------------------+
+| :code:`stride_last`     | bool             | Stride on the final repeated block or all repeated blocks.                                                    | :code:`true` or :code:`false`   |
++-------------------------+------------------+---------------------------------------------------------------------------------------------------------------+---------------------------------+
+| :code:`residual_mode`   | str              | | Type of residual branch to construct.                                                                       | :code:`"add"` or                |
+|                         |                  | | Can be pointwise residual addition or pointwise strided residual attention                                  | :code:`"stride_add"`            |
++-------------------------+------------------+---------------------------------------------------------------------------------------------------------------+---------------------------------+
+
+A Citrinet-512 config might look like below.
+
+.. code-block:: yaml
```

Comment:
Is it necessary to have most of the config here while we can give a reference to the config file?
How about just putting the parts different from Jasper? like just an example of a block?